homeassistant/components/home_connect/entity.py

```diff
@@ -56,3 +56,8 @@ def _handle_coordinator_update(self) -> None:
     def bsh_key(self) -> str:
         """Return the BSH key."""
         return self.entity_description.key
+
+    @property
+    def available(self) -> bool:
+        """Return True if entity is available."""
+        return self.appliance.info.connected is True and self._attr_available
```

Comment:
The coordinator entity has a `available` property connected to the success state of the coordinator last update.
```suggestion
        return (
            self.appliance.info.connected and 
            self._attr_available and
            super().available
        )
```